include/boxed-cpp/boxed.hpp

```diff
@@ -207,19 +207,31 @@ struct hash<boxed::detail::boxed<T, U>>
 } // namespace std
 // {{{ fmtlib integration
 // clang-format off
-#if __has_include(<fmt/format.h>)
+#if __has_include(<format>)
+#include <format>
+// clang-format on
 
+template <typename Type, typename Tag>
+struct std::formatter<boxed::detail::boxed<Type, Tag>>: std::formatter<Type>
+{
+    template <typename FormatContext>
+    auto format(boxed::detail::boxed<Type, Tag> const& val, FormatContext& ctx) const
```

Comment:
Similar to libunicode PR, we can hide the type of context 
```suggestion
    auto format(boxed::detail::boxed<Type, Tag> const& val, auto& ctx) const
```